ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -397,7 +398,9 @@ def pytest_configure(config):
                     del config._metadata[extra_meta]
 
             config._metadata["Test Run Name"] = get_testrun_name()
-            gather_version_info_for_report(config)
+            check_clusters()
+            if ocsci_config.RUN["cephcluster"]:
```

Comment:
Why do you assume that this "cephcluster" value will be there? Couldn't it fail on unknown key?

Reply:
And where is that documented? Please update ocs-ci docs accordingly.

Reply:
Before the condition it set via in line above `check_clusters()` which handles situation where command failed or oc bin is not there. Because it set in run itself I don't think that it need to be documented as it is transparent to the end user.

Reply:
Cool, I don't want to block it.